.buildkite/scripts/triggerJenkinsJob/jenkins.go

```diff
@@ -0,0 +1,96 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package main
```

Comment:
Nit. Move this common logic to a separate package so it is later easier to reuse it in other projects. It may be also nice to have this as a buildkite plugin ;)